_src/static/sass/components/_buttons.scss

```diff
@@ -0,0 +1,371 @@
+// ------------------------------
+// edX Pattern Library: Components - Buttons
+
+// About: Contains base styling for demonstration and utility for buttons.
+// ----------------------------
+
+// #LAYOUT
+// #GLOBALS
+// #BASE (size)
+// #LARGE (size)
+// #SMALL (size)
+// #DEFAULT (style)
+// #DEFAULT-ALT (style)
+// #PRIMARY (style)
+// #SECONDARY (style)
+// #SECONDARY-ALT (style)
+// #LINK (style)
+// #ELEVATED (type)
+// #COMBO (type)
+// #ICONS (type)
+// #OVERLAY (type)
+// ----------------------------
+
+// ----------------------------
+// #LAYOUT
+// ----------------------------
+.example-set {
+    @extend %clear-last-child;
+    padding-bottom: spacing-vertical(base);
+    border-bottom: spacing-vertical(xx-small) solid palette(grayscale, xx-light);
+    margin-bottom: spacing-vertical(base);
+
+    .button-overlay-demo {
+        padding: 2rem;
+        background: palette(grayscale, base);
+    }
+}
+
+
+// ----------------------------
+// #GLOBALS
+// ----------------------------
+.btn {
+    @include transition(
+        color timing(xx-fast) ease-in-out 0s,
+        border-color timing(xx-fast) ease-in-out 0s,
+        background timing(xx-fast) ease-in-out 0s,
+        box-shadow timing(xx-fast) ease-in-out 0s
+    );
+    display: inline-block;
+    border-style: solid;
+    border-radius: 3px;
```

Comment:
Good question. I was actually wondering about this. Pixels allows us to be precise. Not to say that using `rems` wouldn't allow precision, because it's based on a globally set variable, but the pixel conversion would be something like 2.57945 or 3.172938 (as examples) and may not look exactly how we want. If you're okay with that inexactness, I'm okay using `rems` rather than `px` for this.